# header message
Header header

# indicates whether there are hot joints
bool hotJointFound

# contains all hot joints
string[] jointNames
